packages/engine/src/worker/mod.rs

```diff
@@ -532,8 +531,8 @@ impl WorkerController {
         let js_msg = runner_msgs.remove(0);
         tokio::try_join!(
             self.js.send_if_spawned(sim_id, js_msg),
-            /* TODO: self.py.send_if_spawned(msg.sim_id, runner_msgs[1]),
-             * TODO: self.rs.send_if_spawned(msg.sim_id, runner_msgs[2]), */
+            self.py.send_if_spawned(sim_id, runner_msgs.remove(0))
```

Comment:
Is it possible to turn runner_msgs into a slice of length 3 and destructure it, as I think the removes right now are a little unclear, especially since we do a `let js_msg` for JS but not for py.

Something like this?
https://stackoverflow.com/a/67571893/14687716

Perhaps there's a cleaner way

Reply:
I changed it to destructure.

After all the Python init PRs are merged, JS sims will keep working.

Reply:
Will they keep working with each of these PRs though? Otherwise I think we should redirect these into a dev branch that then gets merged in one go

Reply:
Or we should modify the PRs to not break things, with conditional compilation or whatever